TMessagesProj/src/main/res/values/cg_strings.xml

```diff
@@ -225,13 +225,12 @@
     <string name="CG_SyncPins">Sync pinned chats</string>
     <string name="CG_SyncPins_Desc">Disabling this will allow you to have unlimited amount of pinned chats</string>
     <string name="CG_RearCam">Use rear camera</string>
-    <string name="CG_RearCam_Desc">Instead of frontal for video messages</string>
+    <string name="CG_RearCam_Desc">Default to the front camera when starting round video message recording.</string>
     <string name="CG_EnableSwipePIP">Enable Picture-in-Picture for videos</string>
-    <string name="CG_EnableSwipePIP_Desc">by swiping in the video player</string>
```

Comment:
Unclear why this line was deleted entirely *(This should be coordinated with the developer, I think)*

Reply:
I think that an extended description for that is useless, although for a full and complete removal of that string it should also be removed from ChatsPreferencesEntry.kt.